### PR TITLE
cDPE Storage - Firestore Requests

### PR DESCRIPTION
**Pull request by billyjacobson**
Samples:
-Array membership
-Start at snapshot query
-Updating sample data

Tests
README
Composer updates

### CODE REVIEW

File: firestore/composer.json [hunk 1]
```diff
@@ -9,52 +9,54 @@
...
             "src/update_nested_fields.php",
             "src/update_server_timestamp.php",
-            "src/run_simple_transaction.php",
-            "src/return_info_transaction.php",
-            "src/batch_write.php",
-            "src/start_at_field_query_cursor.php",
-            "src/end_at_field_query_cursor.php",
-            "src/paginated_query_cursor.php",
-            "src/multiple_cursor_conditions.php"
+            "src/where_order_by_limit_query.php"
```
[the remark this answers is not in the document]
Comment (billyjacobson) [reply firestore/composer.json]: Yup

**Comment by ryanmats** [comment]
@billyjacobson actually, one more thing to add.

In addition to Array Contains samples, we need samples for updating elements in an array (arrayUnion and arrayRemove). Can you add these?

https://cloud.google.com/firestore/docs/manage-data/add-data#update_elements_in_an_array

Thanks!

**Comment by billyjacobson** [comment]
Added the samples requested @ryanmats

**Comment by ryanmats** [comment]
fantastic - gonna merge.

**Pull request merged**